src/EmbeddedVsix.UnitTests/TypeScriptRuleDefinitions_SanityChecks.cs

```diff
@@ -85,5 +88,8 @@ private static void CheckRules(string language, IEnumerable<RuleDefinition> rule
             !string.IsNullOrEmpty(ruleDefinition.EslintKey) ||
             // Special case - "JavaScript parser failure"
             ruleDefinition.RuleKey.EndsWith(":S2260");
+
+        private static bool HasDefaultParameters(RuleDefinition ruleDefinition) =>
+                ruleDefinition.DefaultParams.Length > 0;
```

Comment:
Rules without default params will have an empty array, rather than null.
We're not expecting `DefaultParams` to be null.